flatpak-lnxlink.yaml

```diff
@@ -0,0 +1,55 @@
+# Generated with flatpak-pip-generator --yaml --output flatpak-lnxlink git+ssh://git@github.com/bkbilly/lnxlink.git@8bf6a5c#egg=lnxlink
+name: python3-lnxlink
+buildsystem: simple
+build-commands:
+  - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+    --prefix=${FLATPAK_DEST} "." --no-build-isolation
+sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/cd/e5/af35f7ea75cf72f2cd079c95ee16797de7cd71f29ea7c68ae5ce7be1eda0/PyYAML-6.0.1.tar.gz
+    sha256: bfdf460b1736c775f2ba9f6a92bca30bc2095067b8a9d77876d1fad6cc3b4a43
+  - type: file
+    url: https://files.pythonhosted.org/packages/1c/d5/c84e1a17bf61d4df64ca866a1c9a913874b4e9bdc131ec689a0ad013fb36/certifi-2024.7.4-py3-none-any.whl
+    sha256: c198e21b1289c2ab85ee4e67bb4b4ef3ead0892059901a8d5b622f24a1101e90
+  - type: file
+    url: https://files.pythonhosted.org/packages/63/09/c1bc53dab74b1816a00d8d030de5bf98f724c52c1635e07681d312f20be8/charset-normalizer-3.3.2.tar.gz
+    sha256: f30c3cb33b24454a82faecaf01b19c18562b1e89558fb6c56de4d9118a032fd5
+  - type: file
+    url: https://files.pythonhosted.org/packages/12/b3/231ffd4ab1fc9d679809f356cebee130ac7daa00d6d6f3206dd4fd137e9e/distro-1.9.0-py3-none-any.whl
+    sha256: 7bffd925d65168f85027d8da9af6bddab658135b840670a223589bc0c8ef02b2
+  - type: file
+    url: https://files.pythonhosted.org/packages/e5/3e/741d8c82801c347547f8a2a06aa57dbb1992be9e948df2ea0eda2c8b79e8/idna-3.7-py3-none-any.whl
+    sha256: 82fee1fc78add43492d3a1898bfa6d8a904cc97d8427f683ed8e798d07761aa0
+  - type: file
+    url: https://files.pythonhosted.org/packages/35/cb/6d564f8a3f25d9516298dce151670d01e43a4b3b769c1c15f40453179cd5/inotify-0.2.10.tar.gz
+    sha256: 974a623a338482b62e16d4eb705fb863ed33ec178680fc3e96ccdf0df6c02a07
+  - type: file
+    url: https://files.pythonhosted.org/packages/11/f1/0f1436d1bb7fae54f557a9965ab2b61163d57853c6f4e2a97236973acd28/jc-1.25.3-py3-none-any.whl
+    sha256: ea17a8578497f2da92f73924d9d403f4563ba59422fbceff7bb4a16cdf84a54f
+  - type: git
+    url: https://git@github.com/bkbilly/lnxlink.git
+    commit: 8dbb784
```

Comment:
If this is the app it should be in the main manifest

Reply:
Added it to the main configuration